spec/rails_helper.rb

```diff
@@ -80,7 +80,8 @@
 VCR.configure do |config|
   config.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
```

Comment:
I am glad we added these lines to VCR and added the api keys to the secure file.